src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -6,39 +6,43 @@ import DescriptionTable from './DescriptionTable';
 import AnswersTable from './AnswersTable';
 import { classes } from './helperFunction';
 
-const CreateConceptForm = props => (
-  <form className="form-wrapper" onSubmit={props.handleSubmit} id="createConceptForm">
-    <div className="concept-form-body">
-      <div className="form-row">
-        <div className="form-group col-md-7">
-          <label htmlFor="uuid">UUID</label>
-          <input
-            type="text"
-            className="form-control"
-            readOnly={props.editable}
-            onChange={props.handleChange}
-            value={props.state.id}
-            name="id"
-            id="uuid"
-            required
-          />
-          <small className="form-text text-muted">
+const CreateConceptForm = (props) => {
+  const {
+    concept, handleAsyncSelectChange, queryAnswers, selectedAnswers, handleAnswerChange,
+  } = props;
+  return (
+    <form className="form-wrapper" onSubmit={props.handleSubmit} id="createConceptForm">
+      <div className="concept-form-body">
+        <div className="form-row">
+          <div className="form-group col-md-7">
+            <label htmlFor="uuid">UUID</label>
+            <input
+              type="text"
+              className="form-control"
+              readOnly={props.editable}
+              onChange={props.handleChange}
+              value={props.state.id}
+              name="id"
+              id="uuid"
+              required
+            />
+            <small className="form-text text-muted">
             Only alphanumeric characters, hyphens, periods, and underscores are allowed.
-          </small>
-        </div>
-        <div className="form-group col-md-3 custom-field">
-          <button
-            type="submit"
-            className="btn btn-sm btn-light"
-            id="toggleUUID"
-            onClick={props.toggleUUID}
-          >
+            </small>
+          </div>
+          <div className="form-group col-md-3 custom-field">
+            <button
+              type="submit"
+              className="btn btn-sm btn-light"
+              id="toggleUUID"
+              onClick={props.toggleUUID}
+            >
```

Comment:
I do not think the changes above should be on this PR.

Reply:
@Tittoh, I think due to changes he took out of this file, after running ESLint, those changes appear and are supposed to be part of this PR.

Reply:
@Tittoh once again, it's the linter issue there.

Reply:
@Tittoh at the moment, until the ticket on linting, is merged, those changes from the linter will continue to happen. I also believe if as long as the linting happens in a file that was being directly worked on...then there should be no issue...